Misc/NEWS.d/next/Library/2024-06-21-06-37-46.gh-issue-120713.WBbQx4.rst

```diff
@@ -0,0 +1,2 @@
+:meth:`!datetime.datetime.strftime` now 0-pads a year <= 999 for the format specifiers ``%Y`` and ``%G`` on Linux.
```

Comment:
Why silence the link?

```suggestion
:meth:`datetime.datetime.strftime` now 0-pads a year <= 999 for the format specifiers ``%Y`` and ``%G`` on Linux.
```

Moreover, please also add a `.. versionchanged` in the `strftime` docs.

Reply:
This is a bug fix, it does not need a `versionchanged`.

Does the reference to `datetime.datetime.strftime` work?

Reply:
> This is a bug fix, it does not need a versionchanged.

Sure, but it is a change of behaviour; it would make sense to document it.

Reply:
Every bugfix is a change of behaviour. If we documented bugfixes in the main documentation, it would be a mess.

Reply:
Of course it is. I'm just saying we do add such directives from time to time; there is precedent. If you are strongly against it, let's not do it.

Reply:
Sounds reasonable not to add bug fixes to the docs. I've reverted the change to the doc then.